### ISSUE TITLE
No DNS record

### ISSUE THREAD

**Issue by userbradley** [alphagov/govuk-kubernetes-cluster-user-docs#91]
The DNS records for this site seem to have been deleted. Was this intentional? 

```shell
➜ nslookup govuk-k8s-user-docs.publishing.service.gov.uk
Server:         8.8.8.8
Address:        8.8.8.8#53

** server can't find govuk-k8s-user-docs.publishing.service.gov.uk: NXDOMAIN
```
```[tasklist]
### Tasks
```

**Comment by sengi** [comment]
Hi! Yes, unfortunately some buffoon (yours truly) mistakenly deleted the magic `gh-pages` branch while cleaning up stale branches, which it turns out is really bad because GitHub pages has several serious flaws with how it handles custom domains. The short story is that custom domain is now burned because GitHub allowed the name to be taken over by spammers.

It'll be back shortly either with a slightly different custom domain, or as part of https://docs.publishing.service.gov.uk/ (where it was supposed to end up eventually anyway).

tl;dr: we screwed up and forgot to protect the magic gh-pages branch 🙃

**Comment by sengi** [comment]
We're back in business https://govuk-kubernetes-cluster-user-docs.publishing.service.gov.uk/

Thanks for getting in touch!
